src/ui/win32/bindings/MemoryViewerControlBinding.cpp

```diff
@@ -264,6 +268,15 @@ void MemoryViewerControlBinding::OnClick(POINT point)
     Invalidate();
 }
 
+void MemoryViewerControlBinding::OnShiftClick(POINT point)
+{
+    OnClick(point);
+    const auto& pEmulatorContext = ra::services::ServiceLocator::Get<ra::data::context::EmulatorContext>();
+    ra::ByteAddress nAddress =
+        pEmulatorContext.ReadMemory(m_pViewModel.GetAddress(), m_pViewModel.GetSize()) & 0xFFFFFF;
```

Comment:
Why are you masking the value? That should be implicit within `ByteAddressFromRealAddress`